source/utils/ResourceManager.cpp

```diff
@@ -49,7 +49,7 @@
 #include <OgreRenderTarget.h>
 
 template<> ResourceManager* Ogre::Singleton<ResourceManager>::msSingleton = 0;
-#if defined(OD_DEBUG)
+#if OGRE_PLATFORM == OGRE_PLATFORM_WIN32 && defined(OD_DEBUG)
```

Comment:
> Because the plugins doesn't seem have the _d suffix on linux systems. (As the debug info is separate). They do in the windows SDK.

I just saw that. Nevermind. Funnily, other's comments have moved in the PR only, so I missed them.